pkg/api/plan/v1alpha1/types.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"sigs.k8s.io/kustomize/kyaml/yaml"
+)
+
+var ResourceMeta = yaml.ResourceMeta{
+	TypeMeta: yaml.TypeMeta{
```

Comment:
Q: Why yaml.TypeMeta / yaml.ObjectMeta vs the apimachinery types?

Reply:
I think either would work here. This just keeps it aligned with how the other types in the project is defined.